frontend/src/schemas/SchemaSelect.tsx

```diff
@@ -36,7 +38,15 @@ export default function SchemaSelect({ entry }: SchemaSelectProps) {
   const activeSchemas = useMemo(() => schemas.filter((schema) => schema.active), [schemas])
   const inactiveSchemas = useMemo(() => schemas.filter((schema) => !schema.active), [schemas])
 
-  const createEntryCardUsingSchema = useCallback(
+  const accessRequestCallback = useCallback(
+    async (newSchema: SchemaInterface) => {
+      setLoading(true)
+      router.push(`/model/${id}/access-request/new?schemaId=${newSchema.id}`)
```

Comment:
Just to future proof ourselves in case any new entry kinds allow access requests I would use the `entry.kind` in the URL:
```
router.push(`/${entry.kind}/${entry.id}/access-request/new?schemaId=${newSchema.id}`)
```